Makefile

```
@@ -77,20 +62,20 @@ bump:
 publish-unpublished: all link
 	$(LERNA) publish from-package
 
-package-lock.json: package.json packages/*/package.json
-	$(NPM) i
+bun.lock: package.json packages/*/package.json
+	bun install
 
 package.json:
 
 install: node_modules
```

checkmake (0.2.2)

[warning] Target "install" should be declared PHONY.

(phonydeclared)